board/pluto/S45msd

```diff
@@ -8,7 +8,7 @@ file=/sys/kernel/config/usb_gadget/composite_gadget/functions/mass_storage.0/lun
 img=/opt/vfat.img
 
 patch_html_page() {
-	LINUX=`uname -a | tr / -`
+	LINUX=`uname -a | tr / -; echo -n "; " $(cat /proc/cpuinfo  | grep processor | wc -l) "core(s)"`
```

Comment:
i think `nproc` may also be available on the device [i only checked the [busybox.config](https://github.com/analogdevicesinc/buildroot/blob/pluto/package/busybox/busybox.config#L273) file]  and if so, it could be used here and would give the same output with just one command

Reply:
I just checked on v0.29 firmware:
```
# nproc
1
```
So, thanks for tip - I will use that.